mne/io/egi/tests/test_egi.py

```diff
@@ -24,17 +24,19 @@ def test_io_egi():
     """Test importing EGI simple binary files"""
     # test default
     with warnings.catch_warnings(record=True) as w:
-        warnings.simplefilter('always', category=RuntimeWarning)
+        warnings.simplefilter('always')
         raw = read_raw_egi(egi_fname, include=None)
         assert_true('RawEGI' in repr(raw))
-        assert_equal(len(w), 1)
-        assert_true(w[0].category == RuntimeWarning)
+        assert_equal(len(w), 2)
+        assert_true(w[0].category == DeprecationWarning)
+        assert_true(w[1].category == RuntimeWarning)
         msg = 'Did not find any event code with more than one event.'
-        assert_true(msg in '%s' % w[0].message)
+        assert_true(msg in '%s' % w[1].message)
 
     include = ['TRSP', 'XXX1']
-    raw = _test_raw_reader(read_raw_egi, False, True, input_fname=egi_fname,
-                           include=include)
+    with warnings.catch_warnings(record=True):  # preload=None
+        raw = _test_raw_reader(read_raw_egi, True, False,
```

Comment:
These readers don't really have a notion of "blocks", but I guess I could leave it in to test slicing. But then why have it be an option at all? If it's to save time, I'll update the docstring to talk about testing slicing, since right now it refers to needing to test `buffer_size_sec`.